app/views/users/show.html.erb

```diff
@@ -1,2 +1,40 @@
-<h1>Users#show</h1>
-<p>Here is a list of users</p>
+<div class="flex flex-col w-full">
+
+<div class="flex items-center justify-between flex-col w-full p-6 bg-white rounded-lg md:flex-row">
+
+  <div class="w-20 mr-2">
+  <%= image_tag 'img2.jpg', alt: 'icon'%>
+  </div>
+
+
+  <div class="flex justify-between w-full">
+    <h2 class="flex text-bold text-2xl"><%= @user.name%></h2>
+    <div>Number of Posts: <%= @user.posts_counter%> </div>
+  </div>
+
+</div>
+
+
+<ul class="w-full max-w-4xl mx-auto p-6 space-y-6">
+  <% @posts.each do |post| %>
+
+  
+    <li class="flex items-center justify-between flex-col w-full p-6 bg-white rounded-lg md:flex-row">
+      <a href="http://127.0.0.1:3000/users/<%=@user.id%>/posts" class="flex w-full">
+        <div class="flex justify-between w-full"><h2 class="flex text-bold text-2xl"><%= post.title%></h2>
+        <ul class="flex justify-between w-48">
+          <li>Comments: <%= post.comments_counter%></li>
+          <li>Likes: <%= post.likes_counter%></li>
+        </ul>
+        </div>
+      </a>
+    </li>
+  <% end %>
+</ul>
+
```

Comment:
- [ ] Uses show page should use the `#recent_post` method from your User model to render the user's 3 most recent post. Kindly address this 💯 👍🏻 

```ruby
  def latest_posts
      Post.where(author_id: id).order(created_at: :desc).limit(3)
   end
```

- Your code should look like this:

   ```ruby
   # app/views/users/show.html.erb
   
   [...]
   
   <% @user.recent_post.each do |post| %>
   
   <h4><%= link_to post.title, user_post_path(post) %></h4>
   <p><%= post.text %></p>
   <p>
   <small>Comments: <%= post.comments_counter || 0 %>,</small> 
   <small>Likes: <%= post.likes_counter || 0 %></small>
   </p>
   
   <% end %>
   
   <button>
   <%= link_to 'See all Post', user_posts_path(@user) %>
   </button>
   ```